biz.aQute.launcher/src/aQute/launcher/plugin/ProjectLauncherImpl.java

```diff
@@ -161,7 +160,7 @@ private LauncherConstants getConstants(Collection<String> runbundles, boolean ex
 
 		if (!exported && !getNotificationListeners().isEmpty()) {
 			if (listenerComms == null) {
-				listenerComms = new DatagramSocket(new InetSocketAddress(InetAddress.getLocalHost(), 0));
+				listenerComms = new DatagramSocket(new InetSocketAddress("localhost", 0));
```

Comment:
Oh yeah, forgot that the launcher is supporting older Java than the main build. 

So this will work on Java 6

```java
listenerComms = new DatagramSocket(new InetSocketAddress(InetAddress.getByName(null), 0));
```

Using `null` for the name represents the loopback address.